pkg/proc/linutil/dynamic.go

```diff
@@ -21,6 +22,24 @@ const (
 	_DT_DEBUG = 21 // DT_DEBUG as defined by SysV ABI specification
 )
 
+func readUintRaw(reader io.Reader, order binary.ByteOrder, ptrSize int) (uint64, error) {
```

Comment:
What's the reason for not using dwarf/util.ReadUintRaw?

Reply:
Because I think `dwarf/util.ReadUintRaw` maybe just used for `dwarf` or something about `dwarf`.  
In fact, I want to put `ReadUintRaw` input one common tool package, but I can't find such package.

Reply:
Need to discuss